designsafe/apps/workspace/admin.py

```diff
@@ -48,10 +61,10 @@ def get_fieldsets(self, request, obj=None):
                 },
             ),
             (
-                "HTML App Body for app_type: HTML",
+                "HTML or External app information",
```

Comment:
This is still only relevant for app_type HTML, right? If so, we should keep that reference somewhere.

Reply:
It is. I have. I improve the reference in 330043ec.

See two fieldsets' names —

- `Tapis App`[^1]
- `HTML or External app`

— match [`app_type` options](https://github.com/DesignSafe-CI/portal/blob/v7.0.9/designsafe/apps/workspace/models/app_entries.py#L161-L164).[^2]

Further improvements welcome.

<details><summary>text-based representation of fieldsets</summary>

> _For the "App type" chosen, edit the corresponding set of fields below_
> **App type**
> **App id**
> 
> <details open><summary>Tapis App</summary>
> 
> _Defines link on CMS app page and display content in Workspace_
> **Version**
> **Href**
> 
> </details>
> 
> <details><summary>HTML or External app</summary>
> 
> _Defines link on CMS app page and display content in Workspace_
> **External HREF**
> **HTML**
> 
> </details>
> <details open><summary>Display</summary>
> 
> _Defines content display on CMS app list and in the Workspace app tray_
> **Label**
> **Description**
> **Enabled**
> **Priority**
> 
> </details>

I kept "HTML or External app" collapsed because it is less common.[^2]

</details>

[^1]: I moved "Tapis App"-specific choices to its own fieldset.
[^2]: I researched making fields dynamic based on `app_type`. That is most simply performed via JavaScript. The only plugin I found to provide the JavaScript looked unreliable. I opted not to write the JavaScript unless someone else agrees to dynamic fields.